app/services/quotable_service.rb

```diff
@@ -0,0 +1,18 @@
+require 'json'
+
+class QuotableService
```

Comment:
This looks great overall. Nice job breaking things out into helper methods. You could also put both your `self.conn and self.parse(api_data)` under the keyword `private`. 
```
your self.quote_search_endpoint(keyword) method

private

your self.conn method 

your self.parse(api_data) method
```